.github/workflows/openqa.yml

```diff
@@ -41,3 +42,29 @@ jobs:
           UEFI_PFLASH_VARS=opensuse-Tumbleweed-x86_64-${{ steps.latest_build.outputs.build }}-textmode@64bit-uefi-vars.qcow2
           BUILD="$GH_REPO.git#$GH_REF" _GROUP_ID="${OPENQA_SCHEDULE_GROUP_ID:-118}"
           CASEDIR="$GITHUB_SERVER_URL/$GH_REPO.git#$GH_REF"
+  clone_mentioned_job:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.opensuse.org/devel/openqa/containers/tumbleweed:client
+    steps:
+      - name: Clone and monitor job mentioned in PR description
+        run: >-
+          expected_url=${OPENQA_HOST:-https://openqa.opensuse.org}/;
+          urls=$(perl -e '$ARGV[0]; while ($ARGV[0] =~ /(\@openqa:?\s+Clone\s+(https?:[^\s]+))/ig) { print("$2\n") if index($2, $ARGV[1]) == 0 }' "$GH_PR_BODY" "$expected_url");
+          if ! [[ $urls ]]; then
+            echo "No test cloned; the PR description does not contain '@openqa: Clone ${expected_url}…'." && exit 0
+          fi;
+          if [[ $DRY_RUN_JOB_IDS ]]; then
+            json=("$DRY_RUN_JOB_IDS");
+          else
+            for url in $urls; do
+              [[ $jobs ]] && jobs+=', ';
+              jobs+="'$url'"
+              json+=("$(openqa-clone-job --apikey "$OPENQA_API_KEY" --apisecret "$OPENQA_API_SECRET" --json-output --skip-chained-deps --within-instance "$url" \
+            BUILD="$GH_REPO.git#$GH_REF" _GROUP_ID="${OPENQA_SCHEDULE_GROUP_ID:-118}" CASEDIR="$GITHUB_SERVER_URL/$GH_REPO.git#$GH_REF")");
+            done;
+          fi;
+          for j in "${json[@]}"; do job_ids+=($(echo "$j" | jq -r '. | values[]')); done;
+          echo "Cloned $jobs into:";
+          for job_id in "${job_ids[@]}"; do echo " - ${expected_url}tests/$job_id"; done;
+          openqa-start cli monitor --host "$expected_url" --apikey "$OPENQA_API_KEY" --apisecret "$OPENQA_API_SECRET" $job_ids
```

Comment:
You can just define a multi-line command section not needing semi-colons (but maybe needing line-continuation with `\` in some cases)

```suggestion
        run: |
          expected_url=${OPENQA_HOST:-https://openqa.opensuse.org}/
…
          for job_id in "${job_ids[@]}"; do echo " - ${expected_url}tests/$job_id"; done
          openqa-start cli monitor --host "$expected_url" --apikey "$OPENQA_API_KEY" --apisecret "$OPENQA_API_SECRET" $job_ids
```

Trying some more ideas:
1. I think sed can be cleaner than Perl here
2. I don't see a need to restrict to `--within-instance`. I would just use `--skip-download --from "$url" --host "$expected_url"`. Would this work?

```suggestion
        run: |
          expected_url=${OPENQA_HOST:-https://openqa.opensuse.org}/
          urls=$(echo "$GH_PR_BODY" | sed -n 's/(\@openqa:?\s+Clone\s+(https?:[^\s]+))/\2/gp')
          if ! [[ $urls ]]; then
            echo "No test cloned; the PR description does not contain '@openqa: Clone ${expected_url}…'." && exit 0
          fi
          if [[ $DRY_RUN_JOB_IDS ]]; then
            json=("$DRY_RUN_JOB_IDS")
          else
            for url in $urls; do
              [[ $jobs ]] && jobs+=', '
              jobs+="'$url'"
              json+=("$(openqa-clone-job --apikey "$OPENQA_API_KEY" --apisecret "$OPENQA_API_SECRET" --json-output --skip-chained-deps --skip-download --from "$url" --host "$expected_url" \
            BUILD="$GH_REPO.git#$GH_REF" _GROUP_ID="${OPENQA_SCHEDULE_GROUP_ID:-118}" CASEDIR="$GITHUB_SERVER_URL/$GH_REPO.git#$GH_REF")")
            done
          fi
          for j in "${json[@]}"; do job_ids+=($(echo "$j" | jq -r '. | values[]')); done
          echo "Cloned $jobs into:"
          for job_id in "${job_ids[@]}"; do echo " - ${expected_url}tests/$job_id"; done
          openqa-start cli monitor --host "$expected_url" --apikey "$OPENQA_API_KEY" --apisecret "$OPENQA_API_SECRET" $job_ids
```